unit_tests/web_tests/test_web_status.py

```diff
@@ -562,7 +562,7 @@ def test_web_status_plot_hide_line(self):
         self.assertFalse(self.page.plot_has_line())
         self.page.click_close_plot()
     
-
+    @unittest.skip("Not Working Issue-437")
```

Comment:
revisit setup and monitor vm situation

Reply:
Will be resolved by hep-gc/uvic-heprc-ansible-playbooks#14